src/tst2.c

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
There are 2 test files, that should be deleted prior merging.
However, they show how UNCRUSTIFY job will fail =)